samples/msal-node-samples/Managed-Identity/ServiceFabric/HttpClient.ts

```diff
@@ -0,0 +1,36 @@
+import https from "https";
```

Comment:
I find it a bit strange that this file is named "HttpClient" as that has a lot of different meaning. Maybe call it "KeyVaultAccessor" or "DownstreamApi" ?